microsoft/testsuites/vm_extensions/AzureMonitorAgentLinux.py

```
@@ -55,7 +55,8 @@ def verify_azuremonitoragent_linux(self, log: Logger, node: Node) -> None:
 
         try:
             # Delete VM Extension if already present
-            extension.delete("AzureMonitorLinuxAgent")
+            extension.delete("Microsoft.Azure.Monitor.AzureMonitorLinuxAgent")
+            is_extension_present = True
         except HttpResponseError as identifier:
             if any(s in str(identifier) for s in ["was not found"]):
```

Please output log in the `else` part as well, even I'm thinking the test case should fail, if it's not "was not found".

---

updated